src/main/java/randoop/generation/UniformRandomMethodSelection.java

```diff
@@ -0,0 +1,29 @@
+package randoop.generation;
+
+import java.util.List;
+import randoop.operation.TypedOperation;
+import randoop.util.Randomness;
+
+/** This class selects a method with uniform probability. */
+public class UniformRandomMethodSelection implements TypedOperationSelector {
+  private final List<TypedOperation> operations;
+
+  /**
+   * Saves a reference to the list of methods under test.
+   *
+   * @param operations methods under test
+   */
+  public UniformRandomMethodSelection(List<TypedOperation> operations) {
+    this.operations = operations;
```

Comment:
Should this make a defensive copy?  (I don't know the answer -- I'm asking.)

Reply:
For some reason, making a copy of the list causes a few system tests to fail - specifically the ones that check coverage goals.  I noted in the documentation the choice to not make a copy.